src/common/meta/src/key/flow/flow_info.rs

```diff
@@ -131,6 +132,12 @@ pub struct FlowInfoValue {
     pub(crate) comment: String,
     /// The options.
     pub(crate) options: HashMap<String, String>,
+    /// The CreateTime.
+    pub(crate) created_time: DateTime<Utc>,
+    /// The UpdateTime.
+    pub(crate) updated_time: DateTime<Utc>,
+    /// The Last Execution Time.
+    pub(crate) last_execution_time: Option<DateTime<Utc>>,
```

Comment:
This execution time get updates too often, so this `last_execution_time` is not suitable in `FlowInfoValue` as updating kv backend is a rather costy operation, consider either remove this column(as this execution time might not be very useful due to flow get executed very often(sub seconds) for now) or update this `last_execution_time` through heart beat just like `FlowStat` did(maybe add a field in `FlowStat`)

Reply:
Thank you very much for your suggestion, I will try to modify it. 

In addition, I would like to ask, does `flush_flow` also rely on Stat to update `last_execution_time`? Or is it updated in real time?

Reply:
Hi, I have updated my code. Using heartbeat to update the flow's `last_exec_time`.

Please review the new code. : )